### DISCUSSION TITLE
OpenVPN

### DISCUSSION THREAD

**Discussion by TripleA150** [keenetic-dev/bypass_keenetic#32] [Q&A] [answered]
Приветствую! Как пустить трафик через openvpn, настроенный на роутере?

**Comment by znetworkx** [comment, marked as answer]
> Приветствую! Как пустить трафик через openvpn, настроенный на роутере?

Трафик через VPN можно запустить достаточно просто:
1. Проверяем в что в bot_config.py есть в строке нужный VPN
2. Настраиваем на роуетере подключение штатными средствами Keenetic к OpenVPN
3. В телеграм боте появится отдельный пункт по добавлению адресов, так же как и на остальные vpn с названием vpn из кинетик. 

Как только вы укажите список адресов и отправите их, маршрутизация будет фильтроваться.

**Reply by TripleA150** [reply]
Спасибо
